plugins/github-deployments/src/components/GithubDeploymentsTable/columns.tsx

```diff
@@ -66,11 +66,14 @@ export function createStatusColumn(): TableColumn<GithubDeployment> {
 export function createCommitColumn(): TableColumn<GithubDeployment> {
   return {
     title: 'Commit',
-    render: (row: GithubDeployment): JSX.Element => (
-      <Link to={row.commit.commitUrl} target="_blank" rel="noopener">
-        {row.commit.abbreviatedOid}
-      </Link>
-    ),
+    render: (row: GithubDeployment): JSX.Element =>
+      row.commit ? (
```

Comment:
Does the render method have to return an element, or could you go with `row.commit && ...` instead? Assuming you get rid of the explicit return type

Reply:
sure that seems to work too, updated